stripe-core/src/test/java/com/stripe/android/core/networking/AnalyticsRequestV2FactoryTest.kt

```diff
@@ -80,7 +80,10 @@ class AnalyticsRequestV2FactoryTest {
         val additionalParam2 = "param2"
         val additionalValue2 = "value2"
         val additionalParam3 = "param3"
-        val additionalValue3 = "value3"
+        val additionalValue3 = mapOf(
+            "nestedParam1" to "nestedValue1",
+            "nestedParam2" to "nestedValue2"
```

Comment:
Updated the test to test ordering, for toString, it's tested through the `URLEncoder.encode` in the test.

For test, trying to plugin the mockable `Clock` class, but since our SDK has min API level 21, we'll need to do the [desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring), which seems a bit overkill given this is just a method call, so will plan to leave as is, wdyt?